pkg/trace/api/api.go

```diff
@@ -294,12 +315,29 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 			r.wg.Done()
 			watchdog.LogOnPanic()
 		}()
-		r.processTraces(ts, traces)
+		entityID := req.Header.Get(headerEntityID)
```

Comment:
we will need to add the `kubernetes_pod://` prefix to `entityID` as in tagger we're converting pod UIDs into entity names
https://github.com/DataDog/datadog-agent/blob/master/pkg/tagger/collectors/kubelet_extract.go#L111
https://github.com/DataDog/datadog-agent/blob/master/pkg/util/kubernetes/kubelet/kubelet_common.go#L26